packages/main/src/themes/CalendarLegendItem.css

```diff
@@ -0,0 +1,73 @@
+.ui5-calendar-legend-item-root {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	width: var(--_ui5-calendar-legend-item-root-width);
+	vertical-align: middle;
+	white-space: pre-line;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	cursor: default;
+}
+
+.ui5-calendar-legend-item-box {
+	display: inline-block;
+	vertical-align: top;
+	box-sizing: border-box;
+	height: 1rem;
+	width: 1rem;
+	margin: var(--_ui5-calendar-legend-item-box-margin);
+}
+
+.ui5-calendar-legend-item-text {
+	display: flex;
+	align-items: center;
+	font-size: 0.875rem;
+	line-height: 1.25rem;
+	color: var(--sapContent_LabelColor);
+	font-weight: normal;
+	font-family: var(--sapFontFamily);
+	font-size: var(--sapFontSmallSize);
+	vertical-align: middle;
+	white-space: pre-line;
+	text-overflow: ellipsis;
+	overflow: hidden;
+	cursor: default;
+}
+
+:host([type="Today"]) .ui5-calendar-legend-item-box {
+	border-color: var(--sapLegend_CurrentDateTime);
+	border: 0.125rem solid var(--sapLegend_CurrentDateTime);
+}
+
+:host([type="Selected"]) .ui5-calendar-legend-item-box {
+	background: var(--sapContent_Selected_Background);
+	border: 0.0625rem solid var(--sapContent_FocusColor);
+}
+
+:host([type="Working"]) .ui5-calendar-legend-item-box {
+	background: var(--sapLegend_WorkingBackground);
+	border: 0.0625rem solid var(--sapContent_ForegroundBorderColor);
+}
+
+:host([type="NonWorking"]) .ui5-calendar-legend-item-box {
+	background: var(--sapLegend_NonWorkingBackground);
+	border: 0.0625rem solid var(--sapContent_ForegroundBorderColor);
+}
+
+.ui5-calendar-legend-item-root:focus {
```

Comment:
I think the outline is sufficient to implement the focus:

```css
.ui5-calendar-legend-item-root:focus {
	outline: var(--_ui5-calendar-legend-item-root-focus-brd);
	border-radius: var(--_ui5-calendar-legend-item-root-focus-brd-radius);
}
```
The pseudo selector has been used when we supported IE as the outline did not follow the border radius.